db/R/query.base.R

```diff
@@ -70,7 +70,7 @@ query.base.con <- function(settings,...){
 ##' @param con database connection object
 ##' @return nothing, as a side effect closes all open connections
 ##' @author Rob Kooper
-##' @export
+##' @export query.close
```

Comment:
This is the only way that I have been able to work around functions with dots in their names getting exported as S3-methods instead of functions.
I'm not sure if we should also be using the `@usage` tag as well. 
http://stackoverflow.com/questions/24594507/exporting-non-s3-methods-with-dots-in-the-name-using-roxygen2-v4